### DISCUSSION TITLE
Is it possible to block the Bottom Sheet from being closed by tapping outside the sheet content area?

### DISCUSSION THREAD

**Discussion by paulfrankallan** [adrielcafe/voyager#343] [Q&A]
My use case is:  Open new screen -> Bottom Sheet pops up to enter some data (over the new screen) -> user can't close sheet to continue unless they complete the data and hit button to update on the Bottom Sheet. They can hit back, which should nav to previous screen not just close sheet. This is my first question here so apologies if it's in the wrong place.

**Comment by Egi10** [comment]
You can recreate the navigator and then set its status to:

```
val sheetState = rememberModalBottomSheetState(
        initialValue = ModalBottomSheetValue.Hidden,
        confirmValueChange = { newValue ->
            newValue != ModalBottomSheetValue.Hidden
        },
        skipHalfExpanded = skipHalfExpanded
    )
```
